### ISSUE TITLE
docs: BlockingIOError(11, 'write could not complete without blocking', 0)

### ISSUE THREAD

**Issue by szcf-weiya** [szcf-weiya/MonotoneSplines.jl#10]
it throws the following error when `include("make.jl")` in the `docs` folder, but it works well if I run the code in the session

```julia
┌ Warning: failed to run `@example` block in src/examples/monofit_mlp_beta_loss.md:62-69
│ ```@example monofit_mlp_beta_loss
│ G, Loss, Loss1, bfun = MonotoneSplines.py_train_G_lambda(y, B, L, nepoch = 0, nepoch0 = 3, K = N,
│             λl = λmin, λu = λmax, gpu_id = -1,
│             nhidden = 1000,
│             λs_opt_train = λs, λs_opt_val = grid_λs,
│             βs_opt_train = βs_mat', βs_opt_val = βs_grid_mat',
│             niter_per_epoch = 200)
│ ```
│   value =
│    PyError ($(Expr(:escape, :(ccall(#= /home/weiya/.julia/packages/PyCall/1gn3u/src/pyfncall.jl:43 =# @pysym(:PyObject_Call), PyPtr, (PyPtr, PyPtr, PyPtr), o, pyargsptr, kw))))) <class 'BlockingIOError'>
│    BlockingIOError(11, 'write could not complete without blocking', 0)
│      File "/home/weiya/GitHub/MonotoneSplines.jl/src/boot.py", line 97, in train_G_lambda
│        pbar0.set_postfix(iter = ii, loss = loss1.item())
│      File "/home/weiya/miniconda3/envs/py3.9torch2.1/lib/python3.9/site-packages/tqdm/std.py", line 1428, in set_postfix
│        self.refresh()
│      File "/home/weiya/miniconda3/envs/py3.9torch2.1/lib/python3.9/site-packages/tqdm/std.py", line 1344, in refresh
│        self.display()
│      File "/home/weiya/miniconda3/envs/py3.9torch2.1/lib/python3.9/site-packages/tqdm/std.py", line 1492, in display
│        self.sp(self.__str__() if msg is None else msg)
│      File "/home/weiya/miniconda3/envs/py3.9torch2.1/lib/python3.9/site-packages/tqdm/std.py", line 347, in print_status
│        fp_write('\r' + s + (' ' * max(last_len[0] - len_s, 0)))
│      File "/home/weiya/miniconda3/envs/py3.9torch2.1/lib/python3.9/site-packages/tqdm/std.py", line 340, in fp_write
│        fp.write(str(s))
│      File "/home/weiya/miniconda3/envs/py3.9torch2.1/lib/python3.9/site-packages/tqdm/utils.py", line 127, in inner
│        return func(*args, **kwargs)
│    
└ @ Documenter.Expanders ~/.julia/packages/Documenter/bYYzK/src/Utilities/Utilities.jl:34
```
